.github/CODEOWNERS

```diff
@@ -119,6 +119,10 @@
 # PRLabel: %Local Endpoints
 /sdk/ml/azure-ai-ml/azure/ai/ml/_local_endpoints/                    @NonStatic2014 @arunsu @stanley-msft @JustinFirsching
 
+# PRLabel: %Job
```

Comment:
This label is way too general.   Can we please find a more specific label that better encapsulates this service?

Reply:
thanks for the comments, will modify in the future if too many pr requests
so far my team would own all pull requests related to jobs

Reply:
This makes repository automation and reporting difficult, as we use labels as identifiers for a lot of logic.   Let's please adjust the label name to something more specific to this service.

Reply:
How about something like `ML Job`?

Reply:
perfect.   We should probably apply the same pattern to "Local Endpoints" as well to keep the pattern predictable